addons/stock/stock.py

```diff
@@ -2571,7 +2571,7 @@ def _get_available_filters(self, cr, uid, context=None):
            :rtype: list of tuple
         """
         #default available choices
-        res_filter = [('none', _('All products')), ('product', _('One product only'))]
+        res_filter = [('none', _('All products')), ('partial', _('Partial')), ('product', _('One product only'))]
```

Comment:
how about `('partial', _('Manual selection of products'))` ?  Keeping in mind that the label is "Inventory of"